octree_web_viewer/src/utils.rs

```diff
@@ -0,0 +1,75 @@
+use actix_web::http::Method;
+use actix_web::{server, HttpRequest, HttpResponse};
+
+use crate::backend::{NodesData, VisibleNodes};
+use crate::backend_error::PointsViewerError;
+use point_viewer::octree;
+use std::path::PathBuf;
+use std::sync::Arc;
+
+const INDEX_HTML: &str = include_str!("../client/index.html");
+const APP_BUNDLE: &str = include_str!("../../target/app_bundle.js");
+const APP_BUNDLE_MAP: &str = include_str!("../../target/app_bundle.js.map");
+
+pub fn index(_req: &HttpRequest) -> HttpResponse {
+    HttpResponse::Ok()
+        .content_type("text/html")
+        .body(INDEX_HTML)
+}
+
+pub fn app_bundle(_req: &HttpRequest) -> HttpResponse {
+    HttpResponse::Ok()
+        .content_type("text/html")
+        .body(APP_BUNDLE)
+}
+
+pub fn app_bundle_source_map(_req: &HttpRequest) -> HttpResponse {
+    HttpResponse::Ok()
+        .content_type("text/html")
+        .body(APP_BUNDLE_MAP)
+}
+
+/// takes  a string input and builds the corresponding octree
+pub fn build_octree(octree_path: &str) -> Arc<octree::Octree> {
```

Comment:
I think instead of having this `utils` methods it would be nice to have a generic octree factory. We could factor out the one inside `SdlViewer` for this, so it can be reused here... What do you think?

Reply:
Good idea, I am looking into this.

Reply:
As discussed offline, for now we just revert the code not to use a `build_octree` helper method. In another PR this will then be replaced with the octree factory `create()` method.

Reply:
done